linkedList.js

```diff
@@ -1,8 +1,123 @@
+/*jshint esversion: 6*/
 /**
  * @name  linkedListGenerator
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
-function linkedListGenerator(){
+ function linkedListGenerator(){
+
+   let linkedList = null;
+
+   function getHead() {
+    return linkedList;
+  }
+
+  function getTail() {
+    let currentNode = linkedList;
+
+    if(!currentNode) {
+      return null;
+
+    } while (currentNode.next) {
+      currentNode = currentNode.next;
+    }
+    return currentNode;
+  }
+
+  function add(value) {
+    let node = {
+      value : value,
+      next: null
+    };
+
+    let currentNode = linkedList;
+
+    if (currentNode === null) {
+      linkedList = node;
+      return node;
+
+    } while (currentNode.next) {
```

Comment:
Fix the formatting on these if statements. The while or whatever comes after should be on a newline.